site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -575,6 +577,36 @@ components:
                      - all
                      - none
 
+      DataVolume:
+         type: object
+         additionalProperties: false
+         properties:
+            "effective-since":
+               type: string
+               format: date-time
+               description: The date-time on which the data volume limit came into effect.
+            "max-bytes":
+               type: integer
+               default: -1
+               description: The maximum number of bytes to be allowed for a tenant for the
+                            defined period. The default value -1 indicates that no limit is set.
+            "period":
+               $ref: '#/components/schemas/Period'
+
+      Period:
+         type: object
+         additionalProperties: false
+         properties:
+            "mode":
+               type: string
+               description: The mode of the data usage caluclation. The supported modes by
+                            the default resource limit checks implementation are `days`
+                            and `monthly`.
+            "no-of-days":
```

Comment:
both properties are required, aren't they?

Reply:
so, at least the *mode* is required then, right? If so, we should mark it accordingly

Reply:
I will mark the _mode_ as required.